CI.hs

```diff
@@ -23,9 +23,10 @@ main = do
 
     appendFile "ghc/hadrian/stack.yaml" $ unlines ["ghc-options:","  \"$everything\": -O0 -j"]
     cmd "stack exec -- ghc-lib-gen ghc --ghc-lib-parser"
-    stackYaml <- readFile' "stack.yaml"
-    writeFile "stack.yaml" $ stackYaml ++ unlines ["- ghc"]
-    tarball <- sDistCreateExtract
+    -- stackYaml <- readFile' "stack.yaml"
```

Comment:
If this is no longer necessary, can we just remove it?

Reply:
✔️